Poi/PoiView.swift

```diff
@@ -149,7 +149,7 @@ public class PoiView: UIView {
         if sender.state == UIGestureRecognizer.State.ended {
             if overlay.center.x < 75 {
                 UIView.animate(withDuration: 0.4, animations: {
-                    card.center = CGPoint(x: card.center.x - 300, y: card.center.y)
+                    card.center = CGPoint(x: card.center.x - 400, y: card.center.y)
```

Comment:
This is fixed value now.
It is better to use the variable according to the screen size.
I don't know accurate value, so I will be glad if you investigate it.